TF2 Rich Presence/locales/Spanish.json

```
@@ -124,13 +124,13 @@
     "2400591970": "Entrenamiento",
     "1589315666": "Recortar/limpiar console.log",
     "8988278560": "Modo de juego desconocido",
-    "1257244521": "Неизвестное имя сервера",
-    "2651621140": "Utilizar las imágenes clásicas del modo de juego en la interfaz gráfica de usuario",
+    "1257244521": "Nombre de Servidor desconocido",
+    "2651621140": "Utilizar las imágenes clásicas del modo de juego en la interfaz",
     "1011879672": "¿Desea abrir la página de descarga?",
     "3846993800": "Sí",
-    "1638865032": "Su instalación de TF2 aún no parece estar configurada correctamente. Para arreglar:",
+    "1638865032": "Su instalación de TF2 aún no parece estar configurada correctamente. Para arreglarla:",
```

This is using "usted" ("su") while other sentences use "tú" (such as the sentences in lines 7 to 12 in `TF2 Rich Presence/locales/Spanish.json`). For consistency this should also use "tú":

```suggestion
    "1638865032": "Tu instalación de TF2 aún no parece estar configurada correctamente. Para arreglarla:",
```